src/content_scripts/applyColorSchemeCss.js

```diff
@@ -12,6 +12,8 @@ const COMMUNICATE_REMOVE_CSS = "removeCss";
 
 // need to save injected CSS, so we can remove it later
 let injectedCss = "";
+// placeholder variable for if the style has failed
+let styledFailed = false;
```

Comment:
Maybe this is a better name?
```suggestion
let applyStyleFailed = false;
```